Product/Production/Deploy/pom.xml

```diff
@@ -11,6 +11,14 @@
   <packaging>pom</packaging>
   <modelVersion>4.0.0</modelVersion>
   
+    <properties>
+        <db.host>localhost</db.host>
```

Comment:
Define db.host and other db config params in pom at CONNECT folder. Remove it from both SoapUI_Test and Deploy.